### PR TITLE
refactor,chore(egen) : refactored code according to template guidelines

### PR DESCRIPTION
**Pull request by Jayakrishna2801**
This PR includes the following updates to the `get_started_with_bqml_training.ipynb` notebook:
 1.Replace REGION with LOCATION.
 2.Removes IS_TESTING condition.
 3.Adds colab enterprise logo with link.
 4.Makes Heading changes according to template guidelines.
 5.Fix unnecessary "``" code highlight usage.
 6.Adds Colab authentication and kernal restart steps.
 7.removed unnecessary code 

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

File: notebooks/official/bigquery_ml/get_started_with_bqml_training.ipynb [hunk 1]
```diff
@@ -80,7 +83,7 @@
         "\n",
         "- BigQuery ML training\n",
         "- Vertex AI model resource\n",
-        "- Vertex AI Vizier\n",
+        "- Vertex AI vizier\n",
```
Comment (kittyabs) [inline notebooks/official/bigquery_ml/get_started_with_bqml_training.ipynb:86]: "Vizier" should be upper case. The product name is "Vertex AI Vizier"

File: notebooks/official/bigquery_ml/get_started_with_bqml_training.ipynb [hunk 2]
```diff
@@ -89,7 +92,7 @@
         "- Evaluate the BigQuery ML model\n",
         "- Export the BigQuery ML model as a cloud model\n",
         "- Upload the exported model as a Vertex AI model resource\n",
-        "- Hyperparameter tune a BigQuery ML model with Vertex AI Vizier\n",
+        "- Hyperparameter tune a BigQuery ML model with Vertex AI vizier\n",
```
Comment (kittyabs) [inline notebooks/official/bigquery_ml/get_started_with_bqml_training.ipynb:95]: "Vizier" should be upper case. The product name is "Vertex AI Vizier"

File: notebooks/official/bigquery_ml/get_started_with_bqml_training.ipynb [hunk 3]
```diff
@@ -404,7 +363,7 @@
       "source": [
         "### Initialize Vertex AI and BigQuery SDKs for Python\n",
         "\n",
-        "Initialize the Vertex AI SDK for Python for your project and corresponding bucket."
+        "Initialize the Vertex AI SDK and BigQuery SDK for Python with your project and the created bucket."
```
Comment (kittyabs) [inline notebooks/official/bigquery_ml/get_started_with_bqml_training.ipynb:366]: change to: (to align with product naming:
- "Initialize the Vertex AI SDK for Python and BigQuery SDK with your project and the created bucket."

File: notebooks/official/bigquery_ml/get_started_with_bqml_training.ipynb [hunk 4]
```diff
@@ -482,14 +441,14 @@
...
         "\n",
-        "Set the pre-built Docker container image for prediction.\n",
+        "Set the prebuilt Docker container image for prediction.\n",
         "\n",
-        "- Set the variable `TF` to the TensorFlow version of the container image. For example, `2-1` would be version 2.1, and `1-15` would be version 1.15. The following list shows some of the pre-built images available:\n",
+        "- Set the variable `TF` to the TensorFlow version of the container image. The following list shows some of the prebuilt images available:\n",
         "\n",
         "\n",
-        "For the latest list, see [Pre-built containers for prediction](https://cloud.google.com/ai-platform-unified/docs/predictions/pre-built-containers)."
+        "For the latest list, see [prebuilt containers for prediction](https://cloud.google.com/ai-platform-unified/docs/predictions/pre-built-containers)."
```
Comment (kittyabs) [inline notebooks/official/bigquery_ml/get_started_with_bqml_training.ipynb:451]: - update this URL:  https://cloud.google.com/ai-platform-unified/docs/predictions/pre-built-containers
- to:                          https://cloud.google.com/vertex-ai/docs/predictions/pre-built-containers

Avoids a redirect

File: notebooks/official/bigquery_ml/get_started_with_bqml_training.ipynb [hunk 5]
```diff
@@ -710,7 +669,7 @@
       "source": [
         "### Export the model from BigQuery ML\n",
         "\n",
-        "The model you trained in BigQuery ML is a TensorFlow model. Next, you export the TensorFlow model artifacts in TF.SavedModel format."
+        "The model you trained in BigQuery ML is a TensorFlow model. Next , export the TensorFlow model artifacts in TF.SavedModel format."
```
Comment (kittyabs) [inline notebooks/official/bigquery_ml/get_started_with_bqml_training.ipynb:672]: Remove extra space after "Next  ,"
"Next, export..."

File: notebooks/official/bigquery_ml/get_started_with_bqml_training.ipynb [hunk 6]
```diff
@@ -782,12 +741,12 @@
       "source": [
         "## Deploy the model\n",
         "\n",
-        "Next, deploy your model for online prediction. To deploy the model, you invoke the `deploy` method, with the following parameters:\n",
+        "Next, deploy your model for online prediction. To deploy the model,  invoke the `deploy` method, with the following parameters:\n",
```
Comment (kittyabs) [inline notebooks/official/bigquery_ml/get_started_with_bqml_training.ipynb:744]: remove extra space: To deploy the model,  invoke
to: To deploy the model, invoke

File: notebooks/official/bigquery_ml/get_started_with_bqml_training.ipynb [hunk 7]
```diff
@@ -920,7 +879,7 @@
       "source": [
         "### Hyperparameter Tune and train a BigQuery ML model\n",
         "\n",
-        "Next, you train a BigQuery ML tabular classification model with hyperparameter tuning using the Vertex AI Vizier service. The hyperparameter settings are specified in the `OPTIONS` statement as follows:\n",
+        "Next, you train a BigQuery ML tabular classification model with hyperparameter tuning using the Vertex AI vizier service. The hyperparameter settings are specified in the `OPTIONS` statement as follows:\n",
```
Comment (kittyabs) [inline notebooks/official/bigquery_ml/get_started_with_bqml_training.ipynb:882]: "Vizier" should be uppercase: "Vertex AI Vizier"

File: notebooks/official/bigquery_ml/get_started_with_bqml_training.ipynb [hunk 8]
```diff
@@ -1040,7 +999,7 @@
       "source": [
         "### Train a BigQuery ML model with Explainability\n",
         "\n",
-        "Next, you train the same BigQuery ML model, but this time you enable Vertex AI Explainability on the model predictions by adding the option:\n",
+        "Next, you train the same BigQuery ML model, this time you enable Vertex AI Explainability on the model predictions by adding the option:\n",
```
Comment (kittyabs) [inline notebooks/official/bigquery_ml/get_started_with_bqml_training.ipynb:1002]: - change: "Vertex AI Explainability"
- to: "Vertex Explainable AI" (this is the product name)

**Review by kittyabs — COMMENTED** [review]
Numerous required edits

**Review by katiemn — APPROVED** [review]

**Pull request merged**